packages/core/examples/loadingSkeletonExample.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as classNames from "classnames";
+import * as React from "react";
+
+import { Classes, LoadingSkeleton, Slider, Switch } from "@blueprintjs/core";
+
+import BaseExample, { handleBooleanChange } from "./common/baseExample";
+
+export interface ILoadingSkeletonState {
+    animated?: boolean;
+    isLoading?: boolean;
+    numBones?: number;
+    randomWidth?: boolean;
+}
+
+export class LoadingSkeletonExample extends BaseExample<ILoadingSkeletonState> {
+    public state: ILoadingSkeletonState = {
+        animated: true,
+        isLoading: true,
+        numBones: 3,
+        randomWidth: false,
+    };
+
+    protected className = "docs-loading-skeleton-example";
+
+    private handleAnimatedChange = handleBooleanChange((animated) => this.setState({ animated }));
+    private handleIsLoadingChange = handleBooleanChange((isLoading) => this.setState({ isLoading}));
+    private handleRandomWidthChange = handleBooleanChange((randomWidth) => this.setState({ randomWidth }));
+
+    public renderExample() {
+        const className = classNames("pt-card", "docs-loading-skeleton-example-box", {
+            "pt-loading": this.state.isLoading,
+        });
+
+        return (
+            <div className={className}>
+                <LoadingSkeleton {...this.state}>
+                    <h5><a href="#">Loading Skeleton</a></h5>
+                    <p>
```

Comment:
Let's remove this `<p>`, it adds unnecessary `margin-bottom` to the block of text:
![image](https://cloud.githubusercontent.com/assets/199754/20983991/69888eb2-bc73-11e6-85c9-05f0d3e0caf4.png)